angular_bind_polymer.js

```diff
@@ -2,6 +2,7 @@ angular.module('eee-c.angularBindPolymer', []).
 directive('bindPolymer', function() {
   return {
     restrict: 'A',
+    scope: {}, /* Scope added here */
```

Comment:
Thanks for looking in to this. I am also new to angular and learning on the go.

Actually, I am trying to pass a callback to the web-component . Something like below.

```
<my-comp name="{{myName}}" action="someAction()"></my-comp>
```
With curly braces i cannot get the callback working since AngularJS does not evaluate the functions from scope. So was trying to use both = and & for models and functions respectively.

The use case is like this. I am trying to create a table web comp and i want to add the ability to add HTMLs to TDs and be able to click on the cell and call the callback.

There may be better ways to accomplish this but at the moment I am not sure how to.